crates/ide-completion/src/completions/postfix.rs

```diff
@@ -284,15 +294,15 @@ fn escape_snippet_bits(text: &mut String) {
     stdx::replace(text, '$', "\\$");
 }
 
-fn include_references(initial_element: &ast::Expr) -> (ast::Expr, ast::Expr) {
+fn include_references(initial_element: &ast::Expr) -> (ast::Expr, String) {
```

Comment:
I had to change this method so that the node we use will be derived from the Semantics.